packages/fxa-graphql-api/src/config.ts

```diff
@@ -44,7 +44,19 @@ const conf = convict({
   contentfulApiKey: {
     doc: 'GraphQL Content API key for Contentful hCMS to fetch RP-provided content',
     format: String,
-    env: 'CONTENTFUL_API_KEY',
+    env: 'CONTENTFUL_GRAPHQL_API_KEY',
+    default: '',
+  },
+  contentfulSpaceId: {
```

Comment:
Now that we have 4 different Contentful-related config values, perhaps it'd be helpful to namespace all of this config under `contenful`, as we have above e.g. with `authServer.url`?